docs/sources/alerting/set-up/configure-high-availability/_index.md

```diff
@@ -60,6 +60,7 @@ Since gossiping of notifications and silences uses both TCP and UDP port `9094`,
    You must have at least one (1) Grafana instance added to the `ha_peers` section.
 1. Set `[ha_listen_address]` to the instance IP address using a format of `host:port` (or the [Pod's](https://kubernetes.io/docs/concepts/workloads/pods/) IP in the case of using Kubernetes).
    By default, it is set to listen to all interfaces (`0.0.0.0`).
+1. Set `[ha_advertise_address]` to `ha_advertise_address = "${POD_IP}:9094"` This is required if the instance doesn't have an IP address that is part of RFC 6890 with a default route.
```

Comment:
I think this description may not be clear to users. `advertise_address` can be an address or a hostname (service name in K8s). This address will be used by other instances in the cluster to communicate with the instance. For example, when instance is behind some NAT (Network Address Translation) where external and internal addresses are differen, whether it's docker network or K8s service, this configuration lets users set the external address which all other instances should use to communicate.

Reply:
Hi @yuri-tceretian this is beyond me in technical terms. Do you think you could help me with some draft content that users will understand and then I can review it? I think that might work better than me stabbing in the dark! :) Thanks!